docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
```

Comment:
Ensure that the `my_db` volume is correctly configured to persist PostgreSQL data. This will help retain your database data even if the container is removed.